client/dashboard/sites/ssh-migration-complete/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { siteBySlugQuery } from '@automattic/api-queries';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import PageLayout from '../../components/page-layout';
+import { SSHMigrationCompleteContentInfo } from '../migration-overview/ssh-migration-complete-content-info';
+
+export default function SSHMigrationComplete( { siteSlug }: { siteSlug: string } ) {
+	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
+
+	return (
+		<PageLayout>
+			<SSHMigrationCompleteContentInfo site={ site } />
```

Comment:
I wonder why we need to extract this component into a separate file instead of just rendering everything here 🤔